tgui/packages/tgui/styles/interfaces/BankingComputer.scss

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author mloccy (https://github.com/mloccy)
+ * @license MIT
+ */
+
+@use '../colors.scss';
+@use '../base.scss';
+
+.headerRow {
```

Comment:
These all pollute the heck out of the top-level namespace and may unintentionally affect other interfaces (either existing or future), make sure to wrap these in some interface-level namespace. Take a look at some of the other interface style overrides for examples; they're all done a bit differently, but broadly you want to make sure some top-level class is applied.